From f8b2483e219897fbf96979b8695de928a5109f69 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 7 Dec 2018 19:15:29 +0100
Subject: [PATCH] Use XDG_CONFIG_HOME as the root of the config path

---
 src/common/config.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/config.cpp b/src/common/config.cpp
index b06e6e40..900e293f 100644
--- a/src/common/config.cpp
+++ b/src/common/config.cpp
@@ -872,7 +872,7 @@ bool parse_ini_file()
 
 	if(!file.is_open())
 	{
-		const char* unix_chr = getenv("HOME");
+		const char* unix_chr = getenv("XDG_CONFIG_HOME");
 		const char* win_chr = getenv("LOCALAPPDATA");
 
 		std::string unix_str = "";
-- 
2.26.0

